QuizApp/QuizApp/User/UserViewController.swift

```diff
@@ -33,6 +106,24 @@ class UserViewController: UIViewController, ConstructViewsProtocol {
         mainView.snp.makeConstraints {
             $0.edges.equalTo(view.safeAreaLayoutGuide)
         }
+
+        label.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(50)
+            $0.leading.equalToSuperview().offset(20)
```

Comment:
All labels should have leading&trailing constraints. Otherwise, they cannot wrap text or have multiline.